src/Sentry/SentryHint.cs

```diff
@@ -80,6 +80,30 @@ public SentryHint(string key, object? value)
         }
     }
 
+    /// <summary>
+    /// Adds a 'byte[]' as attachment to the hind.
+    /// </summary>
+    /// <param name="data">The byte array to be attached</param>
+    /// <param name="fileName">The filename for the attachment</param>
+    /// <param name="type">The type of attachment.</param>
+    /// <param name="contentType">The content type of the attachment.</param>
+    public void AddAttachment(
+        byte[] data,
+        string fileName,
+        AttachmentType type = AttachmentType.Default,
+        string? contentType = null)
+    {
+        if (_options is not null)
```

Comment:
Do we need to check if options is not null here? The options aren't used in this overload right?

Reply:
Fair point! Thanks!